frontend/src/views/Pages/Login.vue

```diff
@@ -88,7 +88,7 @@ export default {
             email: this.form.email,
             password: this.form.password,
           },
-          fetchPolicy: 'no-cache',
+          fetchPolicy: 'network-only',
```

Comment:
Do I understand it right network-only does update the cache so if the user don't have a internet connection can continue to work with the localy stored data!?